src/components/SearchForm/SearchForm.tsx

```diff
@@ -0,0 +1,22 @@
+/* eslint-disable jsx-a11y/label-has-associated-control */
+import React from 'react'
+
+export interface SearchFormProps {
+    LabelContent: string
```

Comment:
Please create semantic props. 
This should be called 'title'.

Reply:
Done